tests/vxingest/utilities/test_backfill_obs_with_rh.py

```diff
@@ -0,0 +1,126 @@
+"""
+    _test for VxIngest backfill_obs_with_rh.py
+"""
+from vxingest.utilities.backfill_obs_with_rh import calc_components
+
+
+def test_calc_components_backfills_rh():
+    """test calculating missing RH"""
+    doc = {
+        "data": {
+            "NZCM": {
+                "Temperature": 25,
+                "DewPoint": 20,
+                "WS": 5,
+                "WD": 180,
+                "WindU": -6.123233995736766e-16,
+                "WindV": 5,
+            },
+            "SUMU": {
+                "Temperature": 30,
+                "DewPoint": 15,
+                "WS": 10,
+                "WD": 270,
+                "WindU": 10,
+                "WindV": 1.8369701987210296e-15,
+            },
+        }
+    }
+    for entries in doc["data"]:
+        assert "RH" not in entries
+    calc_components(doc)
+    assert doc["data"]["NZCM"]["RH"] == 73.78055835142725
```

Comment:
Do we have this much float precision for the calculated `RH`? I wonder if this test is going to be flaky depending on the platform. Ditto for the `["SUMU"]["RH"]` value.

Reply:
I had the same thought, but the tests always passed. I'll go ahead and use the python close enough comparator thing.